ocs_ci/deployment/hub_spoke.py

```
@@ -3013,8 +3015,61 @@ def create_aws_infra(self, timeout=1800):
         except (json.JSONDecodeError, IOError) as e:
             logger.warning(f"Could not read/parse infra output file: {e}")
 
+        self._tag_infra_vpc_with_management_cluster()
+
         return self.output_infra_file
 
+    # Tag key used to identify which management cluster created the HCP infra
+    MGMT_CLUSTER_TAG_KEY = constants.MGMT_CLUSTER_TAG_KEY
+
+    def _tag_infra_vpc_with_management_cluster(self):
```

Ruff (0.16.0)

[warning] Missing return type annotation for private function `_tag_infra_vpc_with_management_cluster`

Add return type annotation: `None`

(ANN202)